errai-cdi-export-apt/src/main/java/org/jboss/errai/apt/internal/export/org_jboss_errai_enterprise_ErraiCdiClientModule__cdi__ExportFile_org_jboss_errai_common_apt_strategies_ErraiExportingStrategies.java

```diff
@@ -0,0 +1,24 @@
+package org.jboss.errai.apt.internal.export;/*
+ * Copyright (C) 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.jboss.errai.cdi.apt.export.ErraiCdiExportingStrategies;
+
+/**
+ * @author Tiago Bento <tfernand@redhat.com>
+ */
+public class org_jboss_errai_enterprise_ErraiCdiClientModule__cdi__ExportFile_org_jboss_errai_common_apt_strategies_ErraiExportingStrategies {
```

Comment:
Yeah, it's a little tricky. `ExportingStrategies` classes have to be exported as a part of a module, but they have to stay on `*-export-apt` module, because they are referenced on the `*ExportFileGenerators`. Annotated classes in `*-export-apt` modules are not exported because these module don't run any annotation processors, so the handwritten export file is necessary to export the `ExportingStrategies` class.

Reply:
Okay, that's fine for now.